### ISSUE TITLE
[Dungeon] Arcatraz

### ISSUE THREAD

**Issue by Silroth** [SunwellWoW/Sunwell-TBC-Bugtracker#431]
[Normal]

Trash:

- [x] Arcatraz Warder not using any of its abilities. Should use Improved Wing Clip on target in melee range then move away and start using Shoot, Arcane Shot and Charged Arcane Shot. EDIT: Does not use Shoot or Charged Arcane Shot, uses Charged Arcane Missiles (wrong spell).

- [x] Arcatraz Defender not using any of its abilities. Should use Immolate and Infected Blood. EDIT: Not using Immolate.

- [x] Protean Horror not using Toothy Bite ability.

- [x] Protean Nightmare not using any of its abilities. Should use Incubation and Gaping Maw.

- [x] Event at start of dungeon with Arcatraz Defenders and Arcatraz Warders fighting Protean Horrors and Nightmares is not working they are just standing still.

- [x] Soul Devourer not using Frenzy ability. EDIT: Using Frenzy incorrectly.

- [x] Sightless Eye not using Sightless Touch.

- [x] Death Watcher not using Death Count ability at 50% health (the one that deals 5k damage if Death Watcher isnt killed under 10 seconds)

- [x] NOTE: Entropic Eye mob missing from dungeon entirely.

- [x] Entropic Eye only uses Tentacle Cleave. EDIT: Uses Tentacle Cleave and Chaos Breath but not the rest of abilities.

- [x] Arcatraz Sentinel Energy Discharge isnt doing damage to players. Should be immune to Taunt and reset aggro every 10 seconds or so.

- [x] Negaton Screamer isnt casting Psychic Scream and he doesnt have the event/spells like with other Voidwalker types that react to schools of magic, more info here https://wow.gamepedia.com/Negaton_Screamer EDIT: Not casting Psychic Scream and should deal Shadow instead of Physical melee damage.

- [x] Negaton Warp-Master isnt using Summon Negaton Field spell.

Pack of Negaton Screamers and Warp-Masters after Zereketh the Unbound should be linked (not 100% sure about this but they way they are positioned now you can pull them 1 by 1 and kill them off easily)

- [x] Following room that has Eredar Deathbringer and 2 Eredar Soul-Eaters has 2 Deatbringers spawned on top of each other, should be only 1 Deathbringer in that room.

- [x] Eredar Deathbringer not casting Diminish Soul. EDIT: Unholy Aura is not working and he does not cast Diminish Soul.

- [x] Eredar Soul-Eater after using Soul Steal the buff with stolen stats doesnt appear on Eredar Soul-Eater.

- [x] Skulking Witch not using any of her abilities except Greater Invisibility. Should use Lash of Pain, AOE Gouge and Chastise. EDIT: Does not use Greater Invisibility.

- [x] Spiteful Temptress not using any of her abilities. Should use Shadow Bolt (this version of Shadow Bolt should explode on impact and deal AOE damage), Domination (Mind Control) on random player on her aggro list and Spiteful Fury which is used on random non-tank player on her aggro list every 10-15 seconds. EDIT: Currently behaves like a caster chain casting Shadow Bolt which shouldnt happen (Shadow Bolt should be used every 10 seconds or so), uses Spiteful Fury but i cannot verify if she uses Domination.

- [x] Unbound Devastator missing his weapon and instead of using his abilities he transforms into a frost troll and gets a debuff called Transport Malfunction? He should use Deafening Roar every 10 seconds (50% uptime) and Devastate. Should be immune to Banish and Enslave. EDIT: Missing weapon and Devastate should deal Arcane instead of Physical damage.

- [x] Ethereum Slayer missing Dual Wield passive ability and not using Evasion ability.

- [x] Ethereum Wave-Caster does not use Sonic Boom or Polymorph.

- [x] Sargeron Archer does not use any of his abilities and when engaged from range do not shoot with their bow they just come to melee range to autoattack 
![image](https://user-images.githubusercontent.com/42102450/45738993-1561a280-bbf2-11e8-9b73-5edef95bcbc2.png)    EDIT: Does not use Immolation Arrow or Scatter Shot (should drop aggro after Scatter Shot on tank) and should behave like a hunter attacking from range.

- [x] Sargeron Hellcaller uses all spells but timers need to be checked. 

- [x] Unchained Doombringer uses his abilities on a wierd timer, Berserker Charge is used once during fight, War Stomp maybe 5-6 times and he used Agonizing Armor only once at around 5% health. EDIT: After engaging uses Berserker Charge and War Stomp only once and never again, Agonizing Armor is used but he doesnt try to stack it (should stack up to 6 times in total).

Bosses:

- [x] Zerekth the Unbound the Unbound Void Zones he summons are autoattacking nearby players for 1 damage (they shouldnt autoattack at all), he is not casting Seed of Corruption.

NOTE: Dalliah and Wrath-Scryer shouldnt walk around but stand in their own corners and stand still, their position right now makes it impossible to pull one of them without pulling the other and they are missing the conversation convo they have when players enter the room or when one of them is pulled.

- [x] Dalliah the Doomsayer Missing Dual Wield passive ability and when engaged uses all active abilities but i am not sure if her Double Attack passive is working (should have 25% chance to proc).

- [x] Wrath-Scryer Soccothrates uses all abilities but for some reason he has 2 Fel Immolation spells on him, first one is correct and deals damage every 3 seconds and other one has Inferno icon and deals damage every 2 seconds (he shouldnt have this one), Felfire Shock is used on wierd timer and when he knocks back and charges Felfire path is spawned in random direction instead of path he charged in. EDIT: Felfire path is not dealing damage.

- Warden Mellichar 
- [x] Twilight Drakonaar missing weapon, when engaged only uses Brood Power: Bronze, he should alternate between other Brood Powers during combat.

- Harbinger Skyriss when engaged uses all abilities properly.

**Comment by Silroth** [comment]
[Heroic]

Trash:

- Arcatraz Warder base damage should be 1700-3300. Should use Shoot ability for 1400-2600. Not using any of its abilities. Should use Improved Wing Clip on target in melee range then move away and start using Shoot, Arcane Shot and Charged Arcane Shot.

- Arcatraz Defender base damage should be 2000-4000. Not using any of its abilities. Should use Immolate for 1200 and ticking 600, Infected Blood ticking for 1575-2025 and Flaming Weapon for 250.

- Protean Horror base damage should be 633-1570. Not using Toothy Bite ability.

- Protean Nightmare base damage should be 3000-6400. Not using any of its abilities. Should use Incubation, Infectious Poison and Gaping Maw ticking for 1425-1575.

Event at start of dungeon with Arcatraz Defenders and Arcatraz Warders fighting Protean Horrors and Nightmares is not working they are just standing still.

- Soul Devourer base damage should be 5200-7000. Fel Breath should deal 2775-3225 damage. Not using Frenzy ability.

- Sightless Eye using Sightless Touch very rarely. Sightless Touch should deal 950-1050.

- Death Watcher base damage should be 5300-6300. Not using Death Count ability at 50% health , should deal 10000-11000 damage. Drain Life should tick for 2000.

NOTE: Entropic Eye mob missing from dungeon entirely.

- Entropic Eye base damage should be 4500-6200. Should use Tentacle Cleave for weapon damage +500, Hex (20yard AoE), Frostbolt/Fireball/Arcane Bolt/Shadow Bolt/Poison Bolt for 2550-3450 (instant), Chaos Breath with one of the following random effects: Corrosive Poison for -5000 armor and ticking 722-928, Fevered Fatigue, Necrotic Poison, Piercing Shadow, Shrink, Wavering Will, Withered Touch ticking for 463 shadow damage and Mana Drain.

- Arcatraz Sentinel base damage should be 4100-5800. Energy Discharge should deal 938 to 1562 damage, Explode for 7600-8100. Should be immune to Taunt and reset aggro every 10 seconds or so.

- Negaton Screamer base damage should be 2750-3850 and dealing Shadow instead of Physical damage. Isnt casting Psychic Scream and he doesnt have the event/spells like with other Voidwalker types that react to schools of magic, more info here https://wow.gamepedia.com/Negaton_Screamer, various Volley Bolts should deal 2125-2875 damage.

- Negaton Warp-Master base damage should be 7000-9400. Isnt using Summon Negaton Field spell. Should heal voidwalkers for 2500 and increasing their damage by 500 stacking up to 10 times a tick.

Pack of Negaton Screamers and Warp-Masters after Zereketh the Unbound should be linked (not 100% sure about this but they way they are positioned now you can pull them 1 by 1 and kill them off easily)

Following room that has Eredar Deathbringer and 2 Eredar Soul-Eaters has 2 Deatbringers spawned on top of each other, should be only 1 Deathbringer in that room.

- Eredar Deathbringer base damage should be 7029-9932. Unholy Aura should tick for 750, not casting Diminish Soul which should be 300 stacking up to 10 times.

- Eredar Soul-Eater base damage should be 6092-8607. After using Soul Steal the buff with stolen stats doesnt appear on Eredar Soul-Eater. Soul Chill should deal 4100-4300 damage.

- Skulking Witch base damage should be 5900-8400. Not using any of her abilities except Greater Invisibility. Should use Lash of Pain for 3325-3675, AOE Gouge and Chastise for 4600-4800.

- Spiteful Temptress base damage should be 4900-7600. Not using any of her abilities. Should use Shadow Bolt for 3150-3850 (this version of Shadow Bolt should explode on impact and deal AOE damage), Domination (Mind Control) on random player on her aggro list and Spiteful Fury which is used on random non-tank player on her aggro list every 10-15 seconds.

- Unbound Devastator base damage should be 6400-9000. Missing his weapon and instead of using his abilities he transforms into a frost troll and gets a debuff called Transport Malfunction? He should use Deafening Roar every 10 seconds (50% uptime) for 2590-3010 and Devastate for 3238-3762. Should be immune to Banish and Enslave.

- Ethereum Slayer base damage should be 3800-5300. Missing Dual Wield passive ability and not using Evasion ability.

- Ethereum Life-Binder base damage should be 2100-3000. Shadow Mend should heal for 9000-11000, Shadow Word: Pain should tick for 857.

- Ethereum Wave-Caster base damage should be 1950-2750. Does not use Sonic Boom or Polymorph. Sonic Boom should deal 3238-3762 damage.

- Sargeron Archer base damage should be 3600-5000. Does not use any of his abilities and when engaged from range do not shoot with their bow they just come to melee range to autoattack 
![image](https://user-images.githubusercontent.com/42102450/45738993-1561a280-bbf2-11e8-9b73-5edef95bcbc2.png)

Should use Shoot ability for 2600-3500, Frost Arrow for 2775-3225, Serpent Sting ticking 694-806, Immolation Arrow for 2775-3225 and Immolation ticking 500-700, Hooked Net for 1110-1290, Scatter Shot and Rapid Fire. 

- Sargeron Hellcaller base damage should be 4100-5600. Uses all spells but timers need to be checked. Hell Rain should tick for 2081-2418, Incinerate for 3700-4300 damage.

- Gargantuan Abyssal base damage should be 6300-10200. Fire Shield for 760-840, Meteor for 17800-19600.

- Unchained Doombringer base damage should be 6600-9400. Uses his abilities on a wierd timer, Berserker Charge is used once during fight, War Stomp maybe 5-6 times and he used Agonizing Armor only once at around 5% health. War Stomp and Berserker Charge should deal higher damage than on normal.


Bosses:

- Zereketh the Unbound base damage should be 3400-4700 and deal Shadow instead of Physical. Right now he is rarely casting Unbound Void Zones (they should deal 2250-2750, damage per tick) and he isnt casting Seed of Corruption which should tick for 600, break at 2500 damage and explode for 2995 damage. Shadow Nova should deal 3700-4500 damage.

- Dalliah the Doomsayer base damage should be 4200-5900. Missing Dual Wield passive ability and when engaged uses all active abilities but i am not sure if her Double Attack passive is working. Gift of the Doomsayer should heal for 4625-5375, Heal for 18500-21500 and should use new ability on heroic which is Shadow Wave for 2250-2750 on random player on her aggro list.

- Wrath-Scryer Soccothrates uses all abilities but for some reason he has 2 Fel Immolation spells on him, first one is correct and deals damage every 3 seconds and other one has Inferno icon and deals damage every 2 seconds (he shouldnt have this one), Felfire Shock is used on wierd timer and when he knocks back and charges Felfire path is spawned in random direction instead of path he charged in. Felfire Shock for 2850-3150 and DoT ticking 950-1050, Felfire trail left on ground for 2200-2400, Fel Immolation for 832-918.


- Warden Mellichar 

Phase-Hunter base damage should be 3200-5400. Back Attack for weapon damage +1850-2150.

Blazing Trickster base damage should be 2800-3800. Fire Shield for +100 fire resistance and 50 damage, Firebolt for 2600-3300.

Akkiris Lightning-Waker base damage should be 3000-4000 (Nature instead of Physical?). Should use Lightning Jump (resets threat), Lightning-Waker's Curse, Magic Grounding, Lightning Discharge for 2600-2700, Spell Shock.

Sulfuron Magma-Thrower base damage should be 4700-6000. Rain of Fire for 1850-2150, Shadow Bolt for 2700-3300.


Twilight Drakonaar base damage should be 4100-5600. Brood Power: Black for 2125-2875, Brood Power: Blue for 1312-1687 damage and 1750-2250 mana burn, Brood Power: Bronze for 788-1012, Brood Power: Green, Brood Power: Red ticking for 525-675.

Blackwing Drakonaar base damage should be 4100-5800. Brood Power: Black for 2125-2875, Mortal Strike for 130% damage, Blast Wave for 3500-4500.


- Harbinger Skyriss base damage should be 4300-6000. Mind Rend ticking 2000, new ability on heroic which is Mana Burn for 1900-2100. Clones should use Mind Rend that ticks for 1250.

**Comment by Riztazz** [comment]
Please re-test after PTR update, gonna happen in a bit, will let you know on discord

**Comment by Wojtyas** [comment]
Everything mentioned here looks fine, was with two groups on Arcatraz and no issues detected so far.
Please create new ticket if new bugs occur.
